static/js/containers/PostPage.js

```diff
@@ -44,19 +53,20 @@ class PostPage extends React.Component {
   }
 
   componentDidUpdate() {
-    const { channels, posts } = this.props;
-    let restStates = [ channels, posts ];
+    const { channels, posts, comments } = this.props;
+    let restStates = [channels, posts, comments];
 
     if (!anyProcessing(restStates) || allLoaded(restStates)) {
       this.updateRequirements();
     }
   }
 
-  renderContents(postID, posts, channelName, channels) {
+  renderContents(postID, posts, channelName, channels, comments) {
     const post = posts.get(postID);
     const channel = channels.get(channelName);
+    const commentTree = comments.get(postID);
```

Comment:
Nitpick: Since `CommentTree` is a component, there could be some confusion with a variable named `commentTree`. I think `commentData` or `commentTreeData` would work here